web/src/lib/InstallerClient.test.js

```diff
@@ -10,16 +10,14 @@ const DBUS_PATH = "/org/opensuse/YaST/Installer";
 const DBUS_IFACE = "org.opensuse.YaST.Installer";
 const LANGUAGE_IFACE = "org.opensuse.DInstaller.Language1";
 const SOFTWARE_IFACE = "org.opensuse.DInstaller.Software1";
+const STORAGE_PROPOSAL_IFACE = "org.opensuse.DInstaller.Storage.Proposal1";
+const STORAGE_ACTIONS_IFACE = "org.opensuse.DInstaller.Storage.Actions1";
```

Comment:
They could be in the proposed _ifaces.js_ module.

Reply:
Something for the `InstallerClient` refactoring.